src/Microsoft.Diagnostics.NETCore.Client/DiagnosticsClient/DiagnosticsClient.cs

```diff
@@ -26,6 +26,11 @@ public sealed class DiagnosticsClient
         {
         }
 
+        public DiagnosticsClient(string address) :
```

Comment:
internal for now? I've been assuming that we'll want to look over a bunch of the reverse server and endpoint work, then make an explicit decision about we feel comfortable marking public.